packages/twitch-eventsub/src/Events/EventSubChannelBanEvent.ts

```diff
@@ -80,4 +86,53 @@ export class EventSubChannelBanEvent {
 	async getBroadcaster(): Promise<HelixUser> {
 		return (await this._client.helix.users.getUserById(this._data.broadcaster_user_id))!;
 	}
+
+	/**
+	 * The ID of the moderator who issued the ban/timeout.
+	 */
+	get moderatorId(): string {
+		return this._data.moderator_user_id;
+	}
+
+	/**
+	 * The name of the moderator who issued the ban/timeout.
+	 */
+	get moderatorName(): string {
+		return this._data.moderator_user_login;
+	}
+
+	/**
+	 * The display name of the moderator who issued the ban/timeout.
+	 */
+	get moderatorDisplayName(): string {
+		return this._data.moderator_user_name;
+	}
+
+	/**
+	 * Retrieves more information about the moderator.
+	 */
+	async getModerator(): Promise<HelixUser> {
+		return (await this._client.helix.users.getUserById(this._data.moderator_user_id))!;
+	}
+
+	/**
+	 * The reason behind the ban.
+	 */
+	get reason(): string {
+		return this._data.reason;
+	}
+
+	/**
+	 * The date and time when the timeout was issued. Will be null for timeouts.
```

Comment:
I think this comment is wrong, see https://dev.twitch.tv/docs/eventsub/eventsub-reference#channel-ban-event

> If it is a timeout, this field shows when the timeout will end.

Reply:
Fixed